docs/explanations/data_collection.rst

```diff
@@ -0,0 +1,89 @@
+What data does Canonical collect from Ubuntu Pro machines?
+**********************************************************
+
+Some system data is sent to Canonical servers for the purpose of delivering
+Ubuntu Pro services in compliance with the terms of the Ubuntu Pro subscriptio
+. This data is sent via a few different methods, depending on the service and
+the purpose of that particular data element.
+
+This document categorises data collection by method of collection.
+
+APT package downloads
```

Comment:
I can be wrong here, but it seems that the `APT packages downloads` and `Livepatch downloads` are not directly tied to data collection per se. It seems more like `data used per service` than a collection of some sorts.

I think there is still value for those sections, but I would not put them under `data collection`. We could create something like `service data needs` for them

Reply:
I think that's a good idea :)

Reply:
Yeah that's a good point, this is data that is sent to canonical servers for the purposes of using the services. I'll rework the structure of this a bit to make that more clear.

Reply:
@s-makin I've restructured the document and headers around this. All the content is the same, but I'm not sure about the new header "Data sent in order to provide service" - do you have any better ideas?